File: storage/src/main/java/com/merfemor/vkwallwatcher/data/VkWallWatchSubscriptionRepository.java
package com.merfemor.vkwallwatcher.data;

import edu.umd.cs.findbugs.annotations.NonNull;
import org.springframework.data.mongodb.repository.MongoRepository;

import java.util.List;

public interface VkWallWatchSubscriptionRepository extends MongoRepository<VkWallWatchSubscription, String> {
    @NonNull
    List<VkWallWatchSubscription> findByChatId(long chatId);
}

File: storage/src/main/java/com/merfemor/vkwallwatcher/data/VkWallWatchSubscriptionRepository.kt
package com.merfemor.vkwallwatcher.data

import org.springframework.data.mongodb.repository.MongoRepository

interface VkWallWatchSubscriptionRepository : MongoRepository<VkWallWatchSubscription, String> {
    fun findByChatId(chatId: Long): List<VkWallWatchSubscription>
}